tensorflow/core/platform/file_system.cc

```diff
@@ -309,7 +309,7 @@ StringPiece FileSystem::Extension(StringPiece path) const {
   StringPiece basename = this->Basename(path);
 
   int pos = basename.rfind('.');
```

Comment:
I would declare `pos` here to be `size_t`. No need for a `static_cast` either since `rfind` returns `size_type` already